app/views/pregnancies/_patient_information.html.erb

```diff
@@ -77,8 +77,28 @@
                        options_for_select(referred_by_options,
                                           pregnancy.referred_by),
                                           autocomplete: 'off' %>
-          <%= f.text_field :special_circumstances, autocomplete: 'Off' %>
-        </div>
+         
+          
+          <%= f.label :special_circumstances, 'Special circumstances' %> 
+          <%= f.form_group :special_circumstances do %>
+            <div class="col-sm-6">
+              <%= f.check_box :special_circumstances, { label: 'Fetal anomaly', name: 'pregnancy[special_circumstances][]', 
```

Comment:
Here's how this is showing up in practice -- they all seem to have similar matching labels - 

```
<div class="checkbox">
  <label for="pregnancy_special_circumstances">
    <input name="pregnancy[special_circumstances][]" type="hidden" value="">
    <input name="pregnancy[special_circumstances][]" type="checkbox" value="Fetal anomaly" id="pregnancy_special_circumstances"> Fetal anomaly
  </label>
</div>
```

So I think what's going on here is that capybara is clicking the associated label, but then getting tripped up because there are six of the same label. (You can kinda see the weirdness if you try clicking on one of them that isn't fetal anomaly -- it'll check or uncheck fetal anomaly regardless.) So I think the challenge here is figuring out how to override the `label_for`.

You might try digging around the bootstrap rails gem  issues -- this looks kinda similar at first glance https://github.com/bootstrap-ruby/rails-bootstrap-forms/issues/178

I'll dig into this with you in a bit.